src/components/metadataFormControls/ModelSingleSelect/BaseModelSingleSelect.tsx

```diff
@@ -32,6 +33,7 @@ export const BaseModelSingleSelect = <
 >({
     available,
     selected,
+    noMatchWithoutFilterText,
```

Comment:
I've decided to generally be a bit explicit in passing this `noMatchWithoutFilterText` prop because it had to be passed through a lot of components in the hierarchy tree, and I didn't want to miss it by passing it with rest props like `...searchableSingleSelectProps`

Reply:
I think that's OK, but not really needed because all the other components extends the level below it, and `spread` down. So it would be quite safe as typescript would've helped you here if you just added it to `SearchableSingleSelect`.